src/AttributeTranslator/ArrayAttributeTranslator.php

```diff
@@ -11,6 +11,10 @@
  */
 final class ArrayAttributeTranslator implements AttributeTranslatorInterface
 {
+    /**
+     * @param array $translations Translations array where each key is an attribute name and the corresponding name is
+     * a translation.
+     */
     public function __construct(
         /**
          * @var array<string,string>
```

Comment:
Since it's a constructor argument and a class attribute at the same time, having docs in "var" is enough (for consistency also).